src/features/cookies/helpers/isConsentChoiceExpired.test.ts

```diff
@@ -18,32 +22,84 @@ mockdate.set(TODAY)
 describe('isConsentChoiceExpired', () => {
   beforeEach(() => storage.clear(COOKIES_CONSENT_KEY))
 
-  it('should be expired if no consent date', async () => {
-    const hasExpired = await isConsentChoiceExpired()
-    expect(hasExpired).toEqual(true)
-  })
+  describe('not expired', () => {
+    it.each([TODAY, LAST_WEEK, THREE_MONTHS_AGO, SIX_MONTHS_MINUS_A_SECOND_AGO])(
+      'should not be expired if user has made choice less than 6 months ago',
+      async (choiceDatetime) => {
+        storage.saveObject(COOKIES_CONSENT_KEY, {
+          choiceDatetime: choiceDatetime.toISOString(),
+        })
 
-  it.each([TODAY, LAST_WEEK, THREE_MONTHS_AGO, SIX_MONTHS_MINUS_A_SECOND_AGO])(
-    'should not be expired if user has made choice less than 6 months ago',
-    async (choiceDatetime) => {
-      storage.saveObject(COOKIES_CONSENT_KEY, {
-        choiceDatetime: choiceDatetime.toISOString(),
-      })
+        const hasExpired = await isConsentChoiceExpired()
+        expect(hasExpired).toEqual(false)
+      }
+    )
 
-      const hasExpired = await isConsentChoiceExpired()
-      expect(hasExpired).toEqual(false)
-    }
-  )
+    describe('storage', () => {
+      beforeEach(() =>
+        storage.saveObject(COOKIES_CONSENT_KEY, {
+          choiceDatetime: SIX_MONTHS_MINUS_A_SECOND_AGO.toISOString(),
+        })
+      )
+
+      it.each(KEYS_TO_REMOVE)(
+        'should not clear optional key "%s" if user has made choice less than 6 months ago',
+        async (key) => {
+          const keyValue = key.toString()
```

Comment:
j'ai eu du mal à comprendre le test

je pense que j'aurais mieux compris si on avait

```suggestion
          const value = 'some value'
```

Reply:
Yes je fais ce changement !